Misc/NEWS.d/next/Library/2022-09-03-09-24-02.gh-issue-96534.EU4Oxv.rst

```diff
@@ -0,0 +1 @@
+Support divert(4) on FreeBSD 14.
```

Comment:
Only on 14?

Reply:
Well, 14 and beyond to be precise.

Reply:
Maybe "Support divert(4) added in FreeBSD 14."?

Reply:
Pushed that!